### ISSUE TITLE
treemacs-select-window seems to be broken

### ISSUE THREAD

**Issue by zoechi** [Alexander-Miller/treemacs#1082]
`treemacs-select-window` always behaved like described - open the treemacs-window if not open and then make it the active window

Since recently, when the treemacs window is already open it doesn't jump to it but jumps to the next window instead (if the treemacs window is not open it behaves correctly)

```
GNU Emacs 29.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.38, cairo version 1.18.0)
[Treemacs] v3.1 (installed 2023.12.30) @ Emacs 29.1
```

**Comment by Alexander-Miller** [comment]
That's actually a feature that was added a while ago. Have a look at `treemacs-select-when-already-in-treemacs`.

**Comment by Alexander-Miller** [comment]
Btw, there's also a handy `treemacs-show-changelog` command.

**Comment by zoechi** [comment]
Thanks a lot for your response.
Good to know about `treemacs-show-changelog`. I didn't expect that but find it helpful.

I couldn't make the `treemacs-select-when-already-in-treemacs` work, but I think that's not what I want anyway.

I always get `../.config/emacs/straight/build/treemacs/treemacs.elc failed to define function treemacs-select-when-already-in-treemacs`

Deleting straight folders and reinstalling, which I already tried before I created this issue, didn't help with `treemacs-select-when-already-in-treemacs`, but `treemacs-select-window` is now again behaving as I know it.

So it's unclear what caused this issue. I'll close and reopen in case I find useful information.

Thanks again for your help :rocket:
